src/ToolBox/superpmi/superpmi-shared/icorjitinfoimpl.h

```diff
@@ -125,6 +125,16 @@
             unsigned*                   offsetAfterIndirection  /* OUT */
             );
 
+    CORINFO_METHOD_HANDLE resolveVirtualMethod(
+        CORINFO_METHOD_HANDLE virtualMethod,
+        CORINFO_CLASS_HANDLE implementingClass
+        );
+
+    CORINFO_METHOD_HANDLE resolveVirtualMethodExact(
```

Comment:
Sure. This is the shim layer header for SPMI, so I'll put something cursory here and something more detailed over in corinfo.h